test/views/PostponedImportVatSpec.scala

```diff
@@ -90,8 +88,9 @@ class PostponedImportVatSpec extends SpecBase {
       }
     }
 
-    "display 'not available' messages correctly when no statements are present and it is after the 14th of the previous month" in new Setup {
-        when(mockDateTimeService.systemDateTime()).thenReturn(LocalDateTime.now().withDayOfMonth(15).minusMonths(1))
+    "display 'not available' messages correctly when no statements are present " +
+      "and it is after the 14th of the previous month" in new Setup {
+        when(mockDateTimeService.systemDateTime()).thenReturn(LocalDateTime.now().withDayOfMonth(DAY_15).minusMonths(ONE_MONTH))
```

Comment:
This line has become very long. Better to split like below

        when(mockDateTimeService.systemDateTime()).
          thenReturn(LocalDateTime.now().withDayOfMonth(DAY_15).minusMonths(ONE_MONTH))